src/types/index.ts

```diff
@@ -7,4 +7,4 @@ export {default as ResizeMode} from './ResizeMode';
 export {default as TextTrackType} from './TextTrackType';
 export {default as ViewType} from './ViewType';
 export * from './video';
-export * from '../specs/VideoNativeComponent';
```

Comment:
I think this was only used for event types so I removed it and added a

```typescript
export type * from '../spec/VideoNativeComponent';
```

in the `events.ts` file. I assumed the default export was not used, if it's used I can restore it like before and add a file that export a dummy object for the web.